asllib/doc/TypeAttributes.tex

```diff
@@ -487,68 +469,66 @@ \subsubsection{Example}
 which ensures that \TypingRuleRef{Structure} terminates\footnote{In mathematical terms,
 this ensures that \TypingRuleRef{Structure} is a proper \emph{structural induction.}}.
 
+\ExampleDef{The Structure of a Type}
+In this example:
+\texttt{type T1 of integer;} is the named type \texttt{T1}
+whose structure is \texttt{integer}.
+
+In this example:
+\texttt{type T2 of (integer, T1);}
+is the named type \texttt{T2} whose structure is (integer, integer). In this
+example, \texttt{(integer, T1)} is non-primitive since it uses \texttt{T1}, which is builtin aggregate.
```

Comment:
grammar:
```suggestion
In this example:
\texttt{type T1 of integer;}, \texttt{T1} is the named type
whose structure is \texttt{integer}.

In this example:
\texttt{type T2 of (integer, T1);}, \texttt{T2}
is the named type whose structure is (integer, integer). In this
example, \texttt{(integer, T1)} is non-primitive since it uses \texttt{T1}, which is builtin aggregate.
```